protocols/radius/src/main/java/org/opennms/protocols/radius/monitor/RadiusAuthMonitor.java

```diff
@@ -168,6 +187,10 @@ public PollStatus poll(MonitoredService svc, Map<String, Object> parameters) {
         String secret = ParameterMap.getKeyedString(parameters, "secret", DEFAULT_SECRET);
         String authType = ParameterMap.getKeyedString(parameters, "authtype", DEFAULT_AUTH_TYPE);
         String nasid = ParameterMap.getKeyedString(parameters, "nasid", DEFAULT_NASID);
+        String innerProtocol = ParameterMap.getKeyedString(parameters, "inner-protocol", DEFAULT_TTLS_INNER_AUTH_TYPE);
```

Comment:
Please document the additional monitor parameters in `opennms-doc/guide-admin/src/asciidoc/text/service-assurance/monitors/RadiusAuthMonitor.adoc`